compiler/z/codegen/ControlFlowEvaluator.cpp

```diff
@@ -323,52 +324,178 @@ generateS390Compare(TR::Node * node, TR::CodeGenerator * cg, TR::InstOpCode::Mne
    return branchOpCond;
    }
 
-static TR::Register*
-xmaxxminHelper(TR::Node* node, TR::CodeGenerator* cg, TR::InstOpCode::Mnemonic compareRROp, TR::InstOpCode::S390BranchCondition branchCond, TR::InstOpCode::Mnemonic moveRROp)
+TR::Register *
+OMR::Z::TreeEvaluator::xmaxxminhelper(TR::Node * node, TR::CodeGenerator * cg)
```

Comment:
I think the original name `xmaxminHelper` should be ok (You missed the caps in Helper when refactoring).